### ISSUE TITLE
spec: colleague experiment — training runs driven from the operator front, detached, remembered [291/S5]

### ISSUE THREAD

**Issue by OriNachum** [agentculture/colleague#295]
# spec: colleague experiment — training runs driven from the operator front, detached, remembered

Part of the #291 holistic scoping (S5). Priority: Wave C. Depends on unsloth-cli S4 (registry/summarize) and S9 (memory scope convention).

## Problem

Fine-tuning on the Spark is operated by hand today: `sloth train` blocks synchronously for the whole run with no job handle, no status, and no memory — the experiment's existence is recorded nowhere an agent recalls. Colleague already solved the long-run problem for its own work items (`work --background`: session-leader detach, machine-readable start payload, file-based status, `clean` reaping) and already remembers lessons to eidetic. The experiment loop should ride the same patterns.

## Owner repo

colleague

## Dependency repos

unsloth-cli (S4 verbs), eidetic-cli (experiment records), lobes-cli (serving the export — existing).

## Proposed interface

A new `colleague experiment` noun (host command, like `flight`), backed by a curated shell-out to the `sloth` CLI (the culture-tool pattern: allow-list exactly `sloth`, identity injected, no torch/unsloth import ever):

- `experiment start --config <toml>` — validates (`sloth validate`), then launches `sloth train` **detached** (the `background.py` one-shot detach precedent: `Popen(start_new_session=True)`, stdio to `.colleague/experiments/<id>/`, machine-readable start payload `{id, pid, log_dir, run_id}`).
- `experiment status [<id>]` — pid liveness + tail of the container log + `sloth runs show` when finished.
- `experiment summarize <id> [--remember]` — `sloth summarize` verbatim; `--remember` upserts an eidetic record (scope per the S9 convention, e.g. `colleague`, type `experiment`) with `{run_id, model, method, dataset sha, final loss, adapter path}`.
- `experiment list` — the registry view (`sloth runs list` + colleague's job dirs).
- `colleague clean` learns to reap dead-pid experiment residue (never a live run).

## CLI examples

```bash
colleague experiment start --config runs/qwen4b-lora.toml --json
colleague experiment status exp-3f2a
colleague experiment summarize exp-3f2a --remember
colleague experiment list --json
```

## Acceptance criteria

- A training run survives the operator closing the session: launched detached with a machine-readable job handle; status queryable mid-run; summary + eidetic record on completion.
- `test_zero_deps.py`/`test_boundary.py` pin: no torch/unsloth import; `experiment` joins the sanctioned subprocess list with a stated reason.
- Missing `sloth` CLI degrades to a clean structured error with a remediation hint (never a traceback).
- An experiment is gradeable through the existing ROI loop (`colleague feedback record <exp-id>`), so experiment quality lands in the same store as work-item quality.

## First implementation step

Failing test: `experiment start` with a stubbed `sloth` on PATH writes the start payload and detaches (no `.wait()`/`.poll()` — the background.py boundary-test pattern extends to the experiment launcher).

## Non-goals

- No scheduler/daemon/queue — one-shot detach + file-based status only (the wrong-abstraction risk named in the frame).
- No dataset preparation (S6) and no sloth-side changes beyond S4.
- No automatic retrain triggers — starting an experiment is always an operator/agent decision.

## Expected size

2–3 PRs (noun + detach launcher; status/list + clean reap; summarize + eidetic remember).

- colleague (Claude)
